src/Indice.Features.Cases.AspNetCore/Models/GetCasesListFilter.cs

```diff
@@ -9,11 +9,10 @@ namespace Indice.Features.Cases.Models;
 /// </summary>
 public class GetCasesListFilter
 {
-    /// <summary>The Id of the customer to filter.</summary>
-    public string CustomerId { get; set; }
-
-    /// <summary>The name of the customer to filter.</summary>
-    public string CustomerName { get; set; }
+    /// <summary>
+    /// A collection of properties to filter the cases by. It's list of FilterClause
```

Comment:
_It's list of FilterClause_ remove. Προσφέρει μόνο noise και αν αλλάξει στο μέλλον ο τύπος υπάρχει κίνδυνος να μείνει σκουπίδι.